src/metric-engine/src/engine/open.rs

```diff
@@ -81,6 +83,7 @@ impl MetricEngineInner {
 
         let mut data_region_options = request.options;
         set_index_options_for_data_region(&mut data_region_options);
+        set_memtable_options_for_data_region(&mut data_region_options);
```

Comment:
Maybe we should merge `set_index_options_for_data_region()` and `set_memtable_options_for_data_region()` into one function.

Reply:
> Maybe we should merge `set_index_options_for_data_region()` and `set_memtable_options_for_data_region()` into one function.

Fixed in 69de0e869